dts/bindings/arm/arm,ethos-u.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Arm Limited and/or its affiliates <open-source-office@arm.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description:
+    The Arm Ethos-U is a micro NPU that enables neural networks to be hardware
+    accelerated on embedded devices. The Ethos-U NPU driver is provided as a
+    Zephyr module that is hooked up into the Tensorflow Lite Micro framework
+    as a 'TFLM custom operation'.
+    
+    This device tree entry defines board specific properties like the register
+    map address and interrupt line, and is therefor expected to be included
+    into the device specific device tree file.
+
+compatible: "arm,ethos-u"
```

Comment:
The pull request is breaking git bisectability. The commit which adds the binding needs to come before the commit which adds the driver using the binding. Please watch https://www.youtube.com/watch?v=mnVNLM7ynMQ for more details.

Reply:
Done. I have switched the order of the patches.